esbm/esbm.html

```diff
@@ -40418,7 +40418,7 @@
 <hr/>
 <p class="ms-0"><strong class="hw" id="pradmiet" style="color: darkblue">Прадме́т</strong> ’канкрэтная матэрыяльная з’ява, рэч’, ’з’ява рэчаіснасці’, ’вучэбная дысцыпліна’ (<v-abbr class="source">ТСБМ</v-abbr>). Праз <v-abbr>рус.</v-abbr> <span style="color: brown">предмет</span> ’тс’ з <v-abbr>польск.</v-abbr> <span style="color: brown">przedmiot</span> ’тс’, якое ў сваю чаргу калькіруе <v-abbr>лац.</v-abbr> <span style="color: brown">objectum</span>: паводле Векслера (Гіст., 151), непасрэднае запазычанне з польскай. Можна дапусціць абодва шляхі запазычання, пра што, мажліва, сведчыць розны націск і адрозненні ў семантыцы, параўн. <span style="color: brown">пре́дміт</span> ’прадмет’ і <span style="color: brown">прідме́т</span>, <span style="color: brown">прядме́т</span> ’рэч, прадмет’ (<v-abbr class="source">Бяльк.</v-abbr>).</p>
 <hr/>
-<p class="ms-0">◎<strong class="hw" id="pradnik" style="color: darkblue">Пра́днік</strong>, <span style="color: brown">пральнай</span>., <span style="color: brown">прсшьнік</span> ’пранік’ (<v-abbr>чырв.</v-abbr>, <v-abbr class="source">З нар. сл.</v-abbr>). Гл. <span style="color: brown">пранік</span>. Форма <span style="color: brown">праднік</span> з <span style="color: brown">праціўнік</span> у выніку дысыміляцыі <span style="color: brown">нн</span> &gt; <span style="color: brown">дп</span>.</p>
+<p class="ms-0">◎<strong class="hw" id="pradnik" style="color: darkblue">Пра́днік</strong>, <span style="color: brown">пра́льнік</span>., <span style="color: brown">пра́ньнік</span> ’пранік’ (<v-abbr>чырв.</v-abbr>, <v-abbr class="source">З нар. сл.</v-abbr>). Гл. <span style="color: brown">пранік</span>. Форма <span style="color: brown">праднік</span> з <span style="color: brown">праньнік</span> у выніку дысыміляцыі <span style="color: brown">нн</span> &gt; <span style="color: brown">нн</span>.</p>
```

Comment:
@375gnu Ці можаш, калі ласка, паправіць:
```suggestion
<p class="ms-0">◎<strong class="hw" id="pradnik" style="color: darkblue">Пра́днік</strong>, <span style="color: brown">пра́льнік</span>., <span style="color: brown">пра́ньнік</span> ’пранік’ (<v-abbr>чырв.</v-abbr>, <v-abbr class="source">З нар. сл.</v-abbr>). Гл. <span style="color: brown">пранік</span>. Форма <span style="color: brown">праднік</span> з <span style="color: brown">праньнік</span> у выніку дысыміляцыі <span style="color: brown">нн</span> &gt; <span style="color: brown">дн</span>.</p>
```
?
У арыгінале `дн`:
<img width="1144" alt="image" src="https://github.com/user-attachments/assets/969785a8-f3b9-4c32-83ef-67b94c7e1aa5">

Reply:
зроблена